src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -521,6 +524,7 @@ export const VaultDepositWithdrawForm = ({
 
   const renderedErrors = errors
     ?.filter((e) => e.long != null)
+    ?.filter((e) => !isSubmitting || e.type.name !== 'error') // hide errors if submitting
```

Comment:
why not combine filters?

Reply:
I realize it's slightly inefficient but I think it's easier to read if I logically separate conditions. Like if I just see `e.long != null && (!isSubmitting || e.type.name !== 'error')` it's not clear what that is checking for or what the sort of logical grouping is.